src/main/java/uk/nhs/digital/nhsconnect/lab/results/configuration/AmqpConfiguration.java

```diff
@@ -0,0 +1,102 @@
+package uk.nhs.digital.nhsconnect.lab.results.configuration;
+
+import org.apache.commons.lang3.StringUtils;
+import org.apache.qpid.jms.JmsConnectionFactory;
+import org.apache.qpid.jms.JmsDestination;
+import org.apache.qpid.jms.message.JmsMessageSupport;
+import org.apache.qpid.jms.policy.JmsDefaultDeserializationPolicy;
+import org.apache.qpid.jms.policy.JmsRedeliveryPolicy;
+import org.springframework.boot.autoconfigure.condition.ConditionalOnMissingBean;
+import org.springframework.context.annotation.Bean;
+import org.springframework.context.annotation.Configuration;
+import org.springframework.jms.support.converter.MappingJackson2MessageConverter;
+import org.springframework.jms.support.converter.MessageConverter;
+
+import javax.jms.ConnectionFactory;
+
+import static org.springframework.util.StringUtils.hasLength;
+
+@Configuration
+@ConditionalOnMissingBean(ConnectionFactory.class)
+public class AmqpConfiguration {
+
+    @Bean
+    public MessageConverter jsonMessageConverter() {
```

Comment:
Is this method being used? IntelliJ doesn't make it clear so I'm not 100% sure if it is

Reply:
Yes, this one can be removed as we are not using it anywhere at the moment. Thanks! 20a5e1be4a78a32cce103b3c68e0a4de4aff5dcc